toxicity/tuning.py

```diff
@@ -34,6 +36,84 @@ def eval_permutation(params, predictor_cls, train_x, train_ys, method='split', n
     return tuple(sorted(params.items())), score
 
 
+def write_results(write_to, scores, predictor_cls):
+    """ Writes experiment results to specified file """
+    with open(write_to, "a") as f:
+            f.write("------------------------------------------------\n")
+            f.write("Model:\t{}\n".format(predictor_cls.name))
+            for params, score in scores:
+                f.write("Score:\t{}\nparams:\t{}\n\n".format(score, dict(params)))
+
+
+def bayesian_optimization(predictor_cls, train_x, train_ys, params, max_iter, max_time, model_type='GP', acquisition_type='EI',
```

Comment:
Should we add a reasonable default argument, either for max_iter or (maybe better) for max_time?